### DISCUSSION TITLE
MVT补帧 经常会生产方框一样的模糊和底部一条模糊

### DISCUSSION THREAD

**Discussion by mustssr** [hooke007/mpv_PlayKit#342] [mpv 通用]
如绿色框框，底部那个一条模糊掉的特别明显，V4/V5一样会出现
环境
WIN11 22H2/CPU5700x/GPU5700xt
![1.webp](https://s2.loli.net/2023/07/24/Rv5IYySXGpjHgcQ.webp)
![2.webp](https://s2.loli.net/2023/07/24/jzuoRcrIlEZgHXh.webp)
![3.webp](https://s2.loli.net/2023/07/24/RK9SDhVmeAOtjyr.webp)

**Comment by hooke007** [comment]
mvt的底边容易出问题，参数不太好调。

而硬字幕和画面混合的地方一直是所有补帧算法的重灾区。

另外你用的是std（现阶段我个人不推荐它），用lq试试

**Reply by mustssr** [reply]
![MPV-00_00_17.872-N0001.webp](https://s2.loli.net/2023/07/25/h4b7BL12jHuxMmw.webp)
我电脑上用LQ会经常出现这种直接边缘变马赛克的情况，比STD还难受😂

**Reply by hooke007** [reply]
你可以试试调参，或者其它方案
